frontend/kesaseteli/youth/public/locales/sv/common.json

```diff
@@ -0,0 +1,51 @@
+{
+  "appName": "SV Nuorten kesäseteli",
+  "header": {
+    "linkSkipToContent": "Gå till innehållet",
+    "menuToggleAriaLabel": "Menu"
+  },
+  "errorPage": {
+    "title": "Ett oförväntat fel skedde",
+    "message": "Något gick fel. Det kan hjälpa, ladda upp sidan på nytt.",
+    "retry": "Ladda upp sidan på nytt.",
+    "logout": "Logga ut"
+  },
+  "404Page": {
+    "pageNotFoundLabel": "Sidan kunde inte hittas",
+    "pageNotFoundContent": "Voi nei, något gick fel. Pröva med att gå tillbaka till ansökan genom framsidan <lnk>här</lnk>"
```

Comment:
I'd rather `link`
```suggestion
    "pageNotFoundContent": "Voi nei, något gick fel. Pröva med att gå tillbaka till ansökan genom framsidan <link>här</link>"
```

Reply:
it cannot be changed because this is used in `Trans` component and `link` is reserved:  https://github.com/i18next/react-i18next/issues/1090#issuecomment-884927848

Reply:
Good to know!